docs/data-sources/cci_namespaces.md

```diff
@@ -0,0 +1,87 @@
+---
+subcategory: "Cloud Container Instance (CCI)"
+---
+
+# huaweicloud_cci_namespaces
+
+Use this data source to obtain CCI namespaces within HuaweiCloud.
+
+## Example Usage
+
+### Filter the list of namespaces by a name keyword
+
+```hcl
+variable "key_word" {}
+
+data "huaweicloud_cci_namespaces" "test" {
+  name = var.key_word
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to obtain the CCI namespace list.
+  If omitted, the provider-level region will be used.
+
+* `type` - (Optional, String) Specifies the CCI namespace type.
+  The valid values are **general-computing** and **gpu-accelerated**.
+
+* `name` - (Optional, String) Specifies the unique name or keyword of the CCI namespace.
+  This parameter can contain a maximum of 63 characters, which may consist of lowercase letters, digits and hyphens.
+
+* `enterprise_project_id` - (Optional, String) Specifies a unique ID in UUID format of enterprise project.
```

Comment:
use “Specifies the enterprise project ID in UUID format .” could be better

Reply:
done